dev.php

```diff
@@ -103,9 +103,12 @@
 					'label' => 'Cite',
 					'attr'  => 'source',
 					'type'  => 'text',
-					'placeholder' => 'Test placeholder',
+					'customAttributes' => array(
```

Comment:
I'd also be open to calling it `attributes` or `field_attributes` if we need to be specific.